_data/utilities.yml

```
@@ -506,18 +506,11 @@ websites:
 
   - name: Verizon
     url: https://www.verizon.com/
-    twitter: VerizonSupport
-    facebook: verizon
     img: verizon.png
-    regions:
-      - us
-
-  - name: Verizon Wireless
-    url: https://www.verizonwireless.com/
-    twitter: VZWSupport
-    img: verizon.png
-    regions:
-      - us
+    tfa:
+      - sms
+      - email
```

The FAQ page also mentions "My Verizon app notifications".
```suggestion
      - email
      - proprietary
```

---

lgtm